docs/configuration/settings.rst

```diff
@@ -122,6 +122,8 @@ Storage
 +------------------------------+-------------------------------+--------------------------------------------------------------------------+
 | kinto.storage_max_backlog    | ``-1``                        | Number of threads that can be in the queue waiting for a connection.     |
 +------------------------------+-------------------------------+--------------------------------------------------------------------------+
+| kinto.storage_strict_json    | ``False``                     | (test only) validate that records are serializable using the json module.|
++------------------------------+-------------------------------+--------------------------------------------------------------------------+
```

Comment:
If it is meant to be used in tests only I don't think it's necessary to document it here. No?